std_msgs/Header header

float64 latitude
float64 longitude
# Height above mean sea level (metres)
float64 altitude

string quality

int8 nb_sat_in_use

# Undulation - the relationship between the geoid and the ellipsoid (m) of the chosen datum
float64 undulation